projects/plugins/jetpack/modules/shortcodes/js/main.js

```diff
@@ -250,9 +250,9 @@
 		},
 	} );
 
-	$( document ).ready( function () {
-		$( '.presentation-wrapper' ).forEach( function () {
-			new Presentation( this );
-		} );
+	addEventListener( 'DOMContentLoaded', () => {
+		[ ...document.getElementsByClassName( 'presentation-wrapper' ) ].forEach(
+			el => new Presentation( el )
+		);
```

Comment:
What about:
```suggestion
		document.querySelectorAll( '.presentation-wrapper' ).forEach( el => {
			// eslint-disable-next-line no-new
			new Presentation( el );
		} );
```
since it doesn't require array spreading?

Reply:
Good idea, looks a lot better.
Why do we need `// eslint-disable-next-line no-new` though? I don't think ESLint complains here.

Reply:
It is being flagged by eslint in my VS Code editor. As I understand it, in our case we explicitly want side effects (attaching event listeners etc), so ignoring the warning makes sense. 
Perhaps my editor is flagging warnings unnecessarily though? Curious to know if you can commit without the comment and if no issues are flagged.

Reply:
When I add it locally, my editor highlights it as unnecessary 🙂 
Let's try to commit without it, and then add it if the checks fail.

Reply:
How odd, looks like it went through just fine.